docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 ---
 sidebar_navigation:
-  title: Nextcloud integration
+  title: Nextcloud integration setup
   priority: 600
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
```

Comment:
Hmm, "market leading collaboration platform". Isn't OpenProject a collaboration platform, too? Shouldn't we focus on the file sharing/serving/hosting bit of Nextcloud?

Reply:
Agreed, I'd recommend the same phrasing as the other doc: "It combines the strengths of [Nextcloud](https://nextcloud.com/), the world’s most-deployed on-premises collaborative file storage platform and [OpenProject](https://www.openproject.org/), the leading free and open source project management and collaboration software."